siliconcompiler/libs/asap7sc7p5t.py

```diff
@@ -82,12 +82,25 @@ def _setup_lib(libname, suffix):
     # Endcap
     lib.add('asic', 'cells','endcap', f"DECAPx1_ASAP7_75t_{suffix}")
 
-    # Techmap
+    # Yosys techmap
     if libname.endswith('rvt'):
         # TODO: write map files for other groups
         lib.add('asic', 'file', 'yosys', 'techmap',
                     libdir + '/techmap/yosys/cells_latch.v')
 
+    # Defaults for OpenROAD tool variables
+    lib.set('asic', 'var', 'openroad', 'place_density', ['0.77'])
```

Comment:
Yes, we definitely need to resolve the grouping. Who "owns" a variable like place_density? Is that a PDK variable, stdlib variable, or user variable. It definitely gets messy if we have multiple setters and we get an ordering issue.